tablecodec/tablecodec_test.go

```diff
@@ -563,20 +547,20 @@ func BenchmarkEncodeValue(b *testing.B) {
 	}
 }
 
-func (s *testTableCodecSuite) TestError(c *C) {
+func TestError(t *testing.T) {
 	kvErrs := []*terror.Error{
 		errInvalidKey,
 		errInvalidRecordKey,
 		errInvalidIndexKey,
 	}
 	for _, err := range kvErrs {
 		code := terror.ToSQLError(err).Code
-		c.Assert(code != mysql.ErrUnknown && code == uint16(err.Code()), IsTrue, Commentf("err: %v", err))
+		require.Truef(t, code != mysql.ErrUnknown && code == uint16(err.Code()), "err: %v", err)
```

Comment:
you can break this assertion into two assertions, one for not equal part, the other for equal part.

Reply:
Thank you.
I've changed